src/Reinforce/RestApi/IComposite.cs

```diff
@@ -0,0 +1,23 @@
+﻿using RestEase;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Reinforce.Constants;
+using Reinforce.RestApi.Models;
+
+namespace Reinforce.RestApi
+{
+    public interface IComposite
+    {
+        [Post("/services/data/{version}/composite")]
+        [Header("Authorization", "Bearer")]
+        Task<TSObject> PostAsync<TSObject>(
```

Comment:
This looks like it might need to be:
`Task<CompositeResponse> PostAsync(`

That, or the generic `TSObject` needs to be passed to `CompositeResponse<TSObject>` and this change to something like:
`Task<CompositeResponse<TSObject>> PostAsync(`

What do you think?